src/css/profile/mobile/common/popup.less

```diff
@@ -11,16 +11,25 @@
 	background-color: @color_popup_bg;
 	border-radius: 26 * @px_base;;
 
-	@media (orientation: portrait) and (max-width: 479px) {
+	// mobile
+	@media (min-width: 0px) and (max-width: 479px) {
 		width: 100%;
 	}
-	@media (orientation: landscape) and (max-height: 479px) {
+	@media (min-height: 0px) and (max-height: 479px) {
```

Comment:
Do you need to check the max-height?

Reply:
I think so, you don't need to check the min-height.

Reply:
yes. I agree comment.
I will remove it.

done.